### ISSUE TITLE
Edit Event Values

### ISSUE THREAD

**Issue by OswArti** [eeglabdevelopers/eeglab#799]
### Description

I am using EEGMATLAB on my MacBook, but the Edit Event Values menu item does not work; the pop-up window does not show. 

<br>
#### Expected behavior:
The pop-up window to show.

<br>
#### Actual behavior:
 the pop-up window does not show

<br>
#### Versions

|   |   |
|----| ---- | 
| OS version     | [Sonoma 14.2.1] |
| Matlab version | [R2024a] |
| EEGLAB version | [v2024.1] |

**Comment by carrie372** [comment]
Same issue using Windows 11

**Comment by arnodelorme** [comment]
You mean EEGLAB, right? Are you using the compile or MATLAB version? Can you share the dataset you are using the exact steps to reproduce the problem?

**Comment by carrie372** [comment]
@arnodelorme I'm using EEGLAB the MATLAB version. I cannot share the dataset however when I use the 2024.0 EEGLAB version on the same dataset the edit event values menu item works properly. Using different datasets the edit event values pop-up window does not work in v2024.1.

**Comment by OswArti** [comment]
Yes, EEGLAB with MATALAB. The problem is while I was doing the examples of the tutorial, for example, Preprocess data

| 
| 
|  | 
Preprocess data

EEGLAB Documentation including tutorials and workshops information
 |

 |

 |




 Oswaldo Artiles 

    El martes, 27 de agosto de 2024, 06:43:24 p.m. GMT+2, Arnaud Delorme ***@***.***> escribió:  
 
 


You mean EEGLAB, right? Are you using the compile or MATLAB version? Can you share the dataset you are using the exact steps to reproduce the problem?

**Comment by arnodelorme** [comment]
Closing for now as we do not have enough info to look into the problem.
